python/lsst/meas/extensions/gaap/_gaap.py

```diff
@@ -551,8 +551,18 @@ def _gaussianizeAndMeasure(self, measRecord: lsst.afw.table.SourceRecord,
             raise measBase.FatalAlgorithmError("No PSF in exposure")
         wcs = exposure.getWcs()
 
-        psfSigma = psf.computeShape(center).getDeterminantRadius()
-        errorCollection = dict()
+        psfShape = psf.computeShape(center)
+        if not (psfShape.getDeterminantRadius() > 0):
+            self.log.debug("PSF shape has non-positive determinant radius at (%f, %f)", center.x, center.y)
```

Comment:
This ticket changes `psfShape` to use `getTraceRadius` always. If so, why is this debug log message relating to `getDeterminantRadius` required? Is the determinant radius used elsewhere, where this logger might be useful, or could it safely be removed?

Reply:
This can be safely removed. I had put that to indicate that the PSF at that location was bad, but I don't think gaap plugin is the place to put that.